### DISCUSSION TITLE
Take the logical reasoning test ✍️

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1211] [Galaxy Trivia] [answered]
Q1. The first question with A as the correct answer is:

    A. Q2
    B. Q3
    C. Q4

Q2. The answer that appears most often is:

    A. C
    B. B
    C. A

Q3. The answer to Q1 is:

    A. B
    B. A
    C. C

Q4. The answer that appears least often is:

    A. A
    B. C
    C. B

**Comment by peterbaccaro** [comment, marked as answer]
Q1. B
Q2. B
Q3. A
Q4. B

**Reasoning**

Q1 can't have A as its answer (otherwise it would contradict itself).

If Q1's answer was C (meaning Q4's answer was A), then Q3's answer would be C. However, Q2's answer should now be A, but this isn't allowed by Q1 (as Q4 is the first answer with A). This is a contradiction.

Therefore, Q1's answer is B (meaning Q3's answer is A).

For Q2's answer to be C, Q4's answer would have to be A, which would contradict Q4. Therefore, Q2's answer is B$ making Q4's answer B.
